omnibus/config/software/python3.rb

```diff
@@ -65,12 +65,18 @@
   end
 
 else
-  dependency "vc_redist_14"
-  default_version "3.7.1"
+  if windows_arch_i386?
+    default_version "3.7.3"
 
-  source :url => "https://s3.amazonaws.com/dd-agent-omnibus/python-windows-#{version}-amd64.zip",
-         :sha256 => "2ebd2eb2155f7c82ec3d2ba5203e0a250b6bbf60c7c5404f1e03f4bc71a096e1"
+    source :url => "http://dbs-laptop/pkg/python-windows-#{version}-x86.zip",
+            :sha256 => "27b8712d98251e861698f3282598001d2e52cbc8dd3743f14313b9c9afedd545"
+  else
+    dependency "vc_redist_14"
```

Comment:
We don't need `vc_redist_14` for 32 bit ?

Reply:
We will, I just haven't done it yet.